src/lib.rs

```diff
@@ -106,7 +116,7 @@ where
     rubber.publish_index(dataset, poi_index).unwrap();
 }
 
-pub fn load_and_index_pois(mut rubber: Rubber, conn: &Connection, dataset: &str) {
+pub fn load_and_index_pois(es: &String, rubber: &mut Rubber, conn: &Connection, dataset: &str) {
```

Comment:
A bit weird to pass both `es` and `rubber`: maybe we could simply pass the connection string and init multiple Rubber when necessary (here and in `index_pois`) ?

Reply:
done